boards/arm/stm32l496g_disco/pinmux.c

```diff
@@ -42,4 +42,4 @@ static int pinmux_stm32_init(struct device *port)
 }
 
 SYS_INIT(pinmux_stm32_init, PRE_KERNEL_1,
-	 CONFIG_PINMUX_STM32_DEVICE_INITIALIZATION_PRIORITY);
+		CONFIG_PINMUX_STM32_DEVICE_INITIALIZATION_PRIORITY);
```

Comment:
Same here, could you put this in its own commit as a whitespace fix unrelated to openocd support?